src/types/bridge.js

```diff
@@ -85,6 +87,19 @@ export interface AccountBridge<T: Transaction> {
     transaction: T
   ): Promise<TransactionStatus>;
 
+  // heuristic that provide the estimated max amount that can be set to a send.
+  // this is usually the balance minus the fees, but it really depends between coins (reserve, burn, frozen part of the balance,...).
+  // it is an heuristic in that this is not necessarily correct and it can be +-delta (so the info can exceed the spendable or leave some dust).
```

Comment:
```suggestion
  // it is a heuristic in that this is not necessarily correct and it can be +-delta (so the info can exceed the spendable or leave some dust).
```